### ISSUE TITLE
Test transaction in README steps does not work

### ISSUE THREAD

**Issue by ra0x3** [FuelLabs/fuel-core#371]
- When pulling the `master` branch of `fuel-core` and trying the steps in the README with a fake transaction, I do not get successful output 
- When following the README steps to start a node (`./target/debug/fuel-core --ip 127.0.0.1 --port 4000`), and submit a test transaction, I get:

```text
➜  fuel-core git:(master) ✗ cargo run --bin fuel-gql-cli -- transaction submit \
"{\"Script\":{\"byte_price\":0,\"gas_price\":0,\"gas_limit\":1000000,\"maturity\":0,\"script\":[80,64,0,202,80,68,0,186,51,65,16,0,36,4,0,0],\"script_data\":[],\"inputs\":[],\"outputs\":[],\"witnesses\":[],\"receipts_root\":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]}}"
    Finished dev [unoptimized + debuginfo] target(s) in 0.34s
     Running `target/debug/fuel-gql-cli transaction submit '{"Script":{"byte_price":0,"gas_price":0,"gas_limit":1000000,"maturity":0,"script":[80,64,0,202,80,68,0,186,51,65,16,0,36,4,0,0],"script_data":[],"inputs":[],"outputs":[],"witnesses":[],"receipts_root":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]}}'`
thread 'main' panicked at 'expected valid endpoint: relative URL without a base', fuel-client/src/main.rs:35:62
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
- Which @Voxelot mentions [here](https://fuellabs.slack.com/archives/C02CX4JVCSJ/p1653767137856479) might be related to requiring a protocol in the default `endpoint` opt
- However, even after making this change from `127.0.0.1:4000` to `http://127.0.0.1:4000` I get another error:
```text
➜  fuel-core git:(master) ✗ cargo run --bin fuel-gql-cli -- transaction submit \
"{\"Script\":{\"byte_price\":0,\"gas_price\":0,\"gas_limit\":1000000,\"maturity\":0,\"script\":[80,64,0,202,80,68,0,186,51,65,16,0,36,4,0,0],\"script_data\":[],\"inputs\":[],\"outputs\":[],\"witnesses\":[],\"receipts_root\":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]}}"
   Compiling fuel-gql-client v0.8.0 (/Users/rashad/Development/repos/fuel-core/fuel-client)
    Finished dev [unoptimized + debuginfo] target(s) in 1.56s
     Running `target/debug/fuel-gql-cli transaction submit '{"Script":{"byte_price":0,"gas_price":0,"gas_limit":1000000,"maturity":0,"script":[80,64,0,202,80,68,0,186,51,65,16,0,36,4,0,0],"script_data":[],"inputs":[],"outputs":[],"witnesses":[],"receipts_root":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]}}'`
thread 'main' panicked at 'invalid transaction json: Error("invalid type: sequence, expected a borrowed string", line: 1, column: 201)', fuel-client/src/main.rs:41:50
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Expected results
- I would expect the fake/test transaction submitted to the started Fuel node to succeed, as it did a few releases ago
- Or, I would expect the documentation to be updated to reflect any changes

**Comment by r-sitko** [comment]
> * When pulling the `master` branch of `fuel-core` and trying the steps in the README with a fake transaction, I do not get successful output
> 
>     * When following the README steps to start a node (`./target/debug/fuel-core --ip 127.0.0.1 --port 4000`), and submit a test transaction, I get:
> 
> 
> ```
> ➜  fuel-core git:(master) ✗ cargo run --bin fuel-gql-cli -- transaction submit \
> "{\"Script\":{\"byte_price\":0,\"gas_price\":0,\"gas_limit\":1000000,\"maturity\":0,\"script\":[80,64,0,202,80,68,0,186,51,65,16,0,36,4,0,0],\"script_data\":[],\"inputs\":[],\"outputs\":[],\"witnesses\":[],\"receipts_root\":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]}}"
>     Finished dev [unoptimized + debuginfo] target(s) in 0.34s
>      Running `target/debug/fuel-gql-cli transaction submit '{"Script":{"byte_price":0,"gas_price":0,"gas_limit":1000000,"maturity":0,"script":[80,64,0,202,80,68,0,186,51,65,16,0,36,4,0,0],"script_data":[],"inputs":[],"outputs":[],"witnesses":[],"receipts_root":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]}}'`
> thread 'main' panicked at 'expected valid endpoint: relative URL without a base', fuel-client/src/main.rs:35:62
> note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
> ```
> 
>     * Which @Voxelot mentions [here](https://fuellabs.slack.com/archives/C02CX4JVCSJ/p1653767137856479) might be related to requiring a protocol in the default `endpoint` opt
> 
>     * However, even after making this change from `127.0.0.1:4000` to `http://127.0.0.1:4000` I get another error:
> 
> 
> ```
> ➜  fuel-core git:(master) ✗ cargo run --bin fuel-gql-cli -- transaction submit \
> "{\"Script\":{\"byte_price\":0,\"gas_price\":0,\"gas_limit\":1000000,\"maturity\":0,\"script\":[80,64,0,202,80,68,0,186,51,65,16,0,36,4,0,0],\"script_data\":[],\"inputs\":[],\"outputs\":[],\"witnesses\":[],\"receipts_root\":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]}}"
>    Compiling fuel-gql-client v0.8.0 (/Users/rashad/Development/repos/fuel-core/fuel-client)
>     Finished dev [unoptimized + debuginfo] target(s) in 1.56s
>      Running `target/debug/fuel-gql-cli transaction submit '{"Script":{"byte_price":0,"gas_price":0,"gas_limit":1000000,"maturity":0,"script":[80,64,0,202,80,68,0,186,51,65,16,0,36,4,0,0],"script_data":[],"inputs":[],"outputs":[],"witnesses":[],"receipts_root":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]}}'`
> thread 'main' panicked at 'invalid transaction json: Error("invalid type: sequence, expected a borrowed string", line: 1, column: 201)', fuel-client/src/main.rs:41:50
> note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
> ```
> 
> Expected results
> 
>     * I would expect the fake/test transaction submitted to the started Fuel node to succeed, as it did a few releases ago
> 
>     * Or, I would expect the documentation to be updated to reflect any changes

Seems that problem is resolved (tested on master commit eec0595).

1. I built fuel-core with `cargo xtask build` command.
2. Run fuel-core `./target/debug/fuel-core --ip 127.0.0.1 --port 4000`
3. Used query from bottom of README page
```
[dev@fedora fuel-core]$ cargo run --bin fuel-gql-cli -- transaction submit \
"{\"Script\":{\"byte_price\":0,\"gas_price\":0,\"gas_limit\":1000000,\"maturity\":0,\"script\":[80,64,0,202,80,68,0,186,51,65,16,0,36,4,0,0],\"script_data\":[],\"inputs\":[],\"outputs\":[],\"witnesses\":[],\"receipts_root\":\"0x6114142d12e0f58cfb8c72c270cd0535944fb1ba763dce83c17e882c482224a2\"}}"
    Finished dev [unoptimized + debuginfo] target(s) in 0.23s
     Running `target/debug/fuel-gql-cli transaction submit '{"Script":{"byte_price":0,"gas_price":0,"gas_limit":1000000,"maturity":0,"script":[80,64,0,202,80,68,0,186,51,65,16,0,36,4,0,0],"script_data":[],"inputs":[],"outputs":[],"witnesses":[],"receipts_root":"0x6114142d12e0f58cfb8c72c270cd0535944fb1ba763dce83c17e882c482224a2"}}'`
0x656da2b4912aa59d2beee8a36fa352e98efdd89a063a556b910ef4b1531d7733
[dev@fedora fuel-core]$ 
```

**Comment by adlerjohn** [comment]
@ra0x3 can you confirm if this was fixed, and if so we can close this issue.
